admin-perspective/4.4/notification-drawer/notification-drawer.md

```diff
@@ -0,0 +1,28 @@
+# Notification Drawer
+
+## Masthead Icon
+![](img/notifications-closed.png)
+- When alerts are firing, a special signifier icon appears in the masthead. Clicking on the icon opens the notification drawer.
```

Comment:
The masthead icon should probably be stronger when there are critical alerts. They require attention, and it would be really easy to miss or ignore the blue dot.

Reply:
I agree. There's an open visual design story to improve the "there's something here" icon. I'll update this when that's finished

Reply:
Yeah, I would expect red or yellow for critical, something subtler for warning only.  But the "you need notification sinks configured" is probably going to be warning level.

Also, we need to make sure we use the right terminology.  We use "sinks" in the alert but you used "receivers" here.

Reply:
Note that we are also using the term "sink" for Knative Event Sources

Reply:
I've been using "Receiver" to align with what the corresponding Alertmanager objects are called https://prometheus.io/docs/alerting/configuration/#receiver 

I believe that's the terminology we use on the page where they're configured as well